src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -91,56 +123,74 @@ class CompanyStep extends React.Component {
         updateReimbursementAccountDraft(value);
     }
 
+    /**
+     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
+     *
+     * @param {String} inputKey
+     * @param {String} value
+     */
+    clearErrorAndSetValue(inputKey, value) {
+        this.setValue({[inputKey]: value});
+        const errors = this.getErrors();
+        if (!errors[inputKey]) {
+            // No error found for this inputKey
+            return;
+        }
+
+        // Clear the existing error for this inputKey
+        const newErrors = {...errors};
+        delete newErrors[inputKey];
+        setBankAccountFormValidationErrors(newErrors);
```

Comment:
NAB, I don't know if we need to handle this here or if we should do it in a future PR to keep things consistent for now, but wanted to call out that we are discussing [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1631304509179900?thread_ts=1631218542.139500&cid=C01GTK53T8Q) that we should only clear errors on submit (possibly on blur in the future) but not on change.

Reply:
Hmm this is how I implemented it on the first step (manually enter Routing and Account number), so it would be consistent with that.

I also think that the slack conversation is focusing on when we "validate". This code is not "validating", is just clearing the error while the user is typing. Even when the error is cleared because the user typed, it could still be "invalid" once he submits.

Reply:
We can sort this out later. Mostly bringing it up since it's a design feature that we need to resolve and get feedback from others on. I agree we can leave things this way for now so that there is consistency.